modules/inputSpinner/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@diana-ui/inputspinner",
```

Comment:
Minor detail: for the expandable panel I named it `"@diana-ui/expandable-panel". So we keep naming consistent, do we want to keep multiple names separated by a hyphen or just have it all together?
I personally think using the hyphen gives a little more legibility.

Reply:
ok, will do